app/screens/LoginScreen.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from "react";
+import { Image, StyleSheet, Text, View } from "react-native";
+import { Button, TextInput } from "react-native-paper";
+
+import colors from "../config/colors";
+import defaultStyles from "../config/styles";
+import Screen from "../components/Screen";
+
+function LoginScreen({ setIsLoggedIn }) {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  function authenticate() {
+    setIsLoggedIn(true);
+    // if(username === 'admin' && password === 'admin')
+    //   setIsLoggedIn(true);
+  }
+
+  return (
+    <Screen style={styles.container}>
+      <Text style={styles.appName}>KiranaShop</Text>
+      <Image style={styles.logo} source={require("../assets/AppLogo.png")} />
+      <View style={styles.form}>
+        <TextInput
+          style={styles.inputTextBox}
+          mode="outlined"
+          label="Username"
+          value={username}
+          onChangeText={(text) => setUsername(text)}
+        />
+        <TextInput
+          style={styles.inputTextBox}
+          mode="outlined"
+          label="Password"
+          value={password}
+          onChangeText={(text) => setPassword(text)}
+        />
+        <Button
+          textColor={"#000"}
+          buttonColor={colors.primary}
+          style={styles.button}
+          mode="contained"
+          onPress={authenticate}
+        >
+          <Text style={defaultStyles.buttonText}>Login</Text>
+        </Button>
+        <Button
+          textColor={"#000"}
+          buttonColor={colors.secondary}
+          style={styles.button}
+          mode="contained"
+          onPress={() => console.log("Pressed")}
```

Comment:
❓ Stray Logging statement or was this needed

Reply:
We can clean that up later as we implement the actual functionality for login.